lib/scheduler.js

```diff
@@ -8,11 +8,11 @@ var { storageManager } = require('./storage-manager.js');
 var { toolbarButton } = require('./toolbar-button.js');
 var { utils } = require('./utils.js');
 
+var destroyTimer;
```

Comment:
Some of this is left over from the previous iteration and some, because I need to learn more about `let` etc. I definitely want to move over to ES6 as much as possible but, want to keep the changes as incremental as possible.